qa/tasks/cephfs/test_volumes.py

```diff
@@ -50,18 +50,24 @@ def _create_or_reuse_test_volume(self):
 
     def  _get_subvolume_group_path(self, vol_name, group_name):
         args = ("subvolumegroup", "getpath", vol_name, group_name)
-        path = self._fs_cmd(*args)
-        # remove the leading '/', and trailing whitespaces
-        return path[1:].rstrip()
+        try:
+            path = self._fs_cmd(*args)
```

Comment:
Shouldn't this command throw some kind of error if the subvolumegroup does not exist? What is it returning now?

Reply:
Now it is Crash. 

From a `test_` function, pass a  `group_name` that doesn't exist. These two functions assume that the subvolume and subvolumegroup exist.